react/ui-components/src/hoc/FormComposerV2.js

```
@@ -377,6 +377,118 @@ export const FormComposer = (props) => {
+          </span>
+          <div style={{ flex: 1, height: "1px", backgroundColor: "#D6D5D4" }}></div>
+        </div>
+      )}
+      {props.loginWithMicroSoft && (
+        <div
+          className="microsoft-login-btn"
+          style={{
+            margin: "0 auto",
+            cursor: "pointer",
+            backgroundColor: "#f47738",
+            border: "1px solid #f47738",
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)